kotlin/workflow-rx2/src/main/java/com/squareup/workflow/rx2/RxWorkers.kt

```diff
@@ -24,20 +24,20 @@ import kotlinx.coroutines.reactive.openSubscription
 import kotlinx.coroutines.rx2.await
 import kotlinx.coroutines.rx2.openSubscription
 
-inline fun <reified T : Any> Observable<T>.asWorker(key: String = ""): Worker<T> =
+inline fun <reified T : Any> Observable<out T>.asWorker(key: String = ""): Worker<T> =
```

Comment:
Is this really necessary? I remember there are problems with some operators. Would this compile?
```kotlin
Observable.just("2").startWith("1").asWorker(..)
```

Reply:
Yep